### PR TITLE
ESY-6972 Decorate use case card with supported payment method(s)

### PR DESCRIPTION
**Pull request by m-lilja**
ESY-6972 Decorate use case card with supported payment method(s)
https://payexjira.atlassian.net/browse/ESY-6972

Also fixed error in code example

### CODE REVIEW

**Review by arebra — APPROVED** [review]
Looks good to me!

File: _includes/use-case-recurring.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ api-supported-versions: 3.x/2.0{% endcapture %}
           "expiryDate": "12/2033"
         }
        // ... other details
-    }
+    ]
```
Comment (m-lilja) [inline _includes/use-case-recurring.md:119]: Corrected typo causing error in code example

File: checkout-v3/index.md [hunk 2]
```diff
@@ -78,7 +82,13 @@ this section. We recommend [getting started here][get-started].
          </span>
          <span class="cards-content">
             <span class="h4">One-Click Payments</span>
-            <span><p>Save your customer's details and make their checkout faster.</p>
+            <span>
+               <p>Save your customer's details and make their checkout faster.</p>
+               <span class="badge badge-gray-light">Card</span>
```
Comment (m-lilja) [inline checkout-v3/index.md:87]: The code `<span class="badge badge-gray-light">Payment method name</span>` result in a grey badge on the card, shown in the screenshot below

image

File: checkout-v3/index.md [hunk 3]
```diff
@@ -126,8 +136,6 @@ below to read more." %}
 [swp-logo]: /assets/img/swedbank-pay-vertical-black.svg
 [swish-logo]: /assets/img/icon-swish-simple.svg
 [trustly-logo]: /assets/img/icon-trustly-simple.svg
-[wcag-presentation]: https://www.swedbankpay.com/information/wcag
-[paid]: /checkout-v3/features/technical-reference/resource-sub-models#paid
```
Comment (m-lilja) [inline checkout-v3/index.md:130]: Autoremoved, seems these links are not in use and were therefore removed automatically

**Pull request merged**